project_organization.md

```diff
@@ -55,6 +55,26 @@ OTIAB members have the following primary responsibilities:
 - Provide feedback about possible priorities and/or specific proposals from the OTSC
 - Represent OpenTracing’s interests within their own organizations
 
+## OpenTracing Working Groups
+
+Working Group members meet at regularly to discuss and define the OpenTracing project structure, as well as perform day-to-day project management duties.
+
+### Language Working Group
```

Comment:
So to be clear, there is one group that manages all languages? Or one group per language?

The name of this working group is important, and after thinking about it I'm not sure "Language" is the right word, esp if it cuts across all languages. Maybe "API and Integrations Working Group"??? (To distinguish from "documentation") Or "API and Ecosystem Working Group" since that's what's being "managed" in the description below?

Reply:
@bhs @tedsuo I prefer one group per language, if we have enough supporter.

Reply:
Having discussed it with many people, I suggest we start with one group, splitting out later once we understand how that will work. We can update the project organization again when that occurs.

Reply:
I mentioned in another comment, but the reason we want to start with a single group is to identify and work on cross-language issues first.